Lib/Inc/CParser.h

```diff
@@ -0,0 +1,91 @@
+/**
+ * @file CParser.h
```

Comment:
This parser deals with JSON, so it should be reflected in the name. Otherwise, parser that handles plain strings will have to have some odd name.